src/components/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
+
+export const HomePage: FC = () => (
+  <h1 className="title">Home page</h1>
+);
```

Comment:
For example, here we can use this div conatiner, to wrap our page. 
It is not correct when you render the page and return only the title

```suggestion
export const HomePage: FC = () => (
  <div className="container">
    <h1 className="title">Home page</h1>
  </div>
);
```